build-support/travis/generate_travis_yml.py

```diff
@@ -8,13 +8,18 @@
 import pystache
 
 
-num_integration_shards = 20
+num_py3_integration_shards = 13
+num_py2_blacklist_integration_shards = 7
+num_cron_integration_shards = 20
 
 
 HEADER = """
 # GENERATED, DO NOT EDIT!
 # To change, edit build-support/travis/travis.yml.mustache and run
 # ./pants --quiet run build-support/travis:generate_travis_yml > .travis.yml
+#
+# Tip: Copy the generated `.travis.yml` into https://yamlvalidator.com to validate the YAML
```

Comment:
Amazing!

Reply:
As a quick check, you can also run `yq .travis.yml`, which will echo the yaml if it validates, but give a completely useless error message if it doesn't...